tests/acceptance/features/lib/AdminSharingSettingsPage.php

```diff
@@ -338,6 +351,175 @@ public function enableExcludeGroupFromSharing(Session $session) {
 			$this->excludeGroupFromSharingCheckboxId
 		);
 	}
+	/**
+	 * @return string
```

Comment:
```suggestion
	 * @return NodeElement|null
```
same for similar functions and functions that use these kind of functions